Move Fastly custom cache configuration to new topic: (#7869)
magento/devdocs: src/cloud/cdn/configure-fastly-customize-cache.md

![Configure purge options]({{ site.baseurl }}/common/images/cloud/cloud_fastly-purgeoptions.png){:width="650px"}

{:.procedure}
To configure Fastly purge options:

1. In the *Fastly Configuration* section, expand **Advanced**.

1. All purge options display. Select "Yes" per purge option to enable automatic purging. Select "No" to disable automatic purging, allowing you to manually purge caches through the Cache Management page.

1. Click **Save Config** at the top of the page.

1. After the page reloads, click **Upload VCL to Fastly** in the *Fastly Configuration* section.

For more information, see [the Fastly configuration options](https://github.com/fastly/fastly-magento2/blob/21b61c8189971275589219d418332798efc7db41/Documentation/CONFIGURATION.md#further-configuration-options).

## Configure GeoIP handling

The Fastly module includes GeoIP handling to automatically redirect visitors or provide a list of stores matching their obtained country code. If you already use a Magento extension for GeoIP handling, you may need to verify the features with Fastly options.

> 

Reviewer: Missing procedure title?

> {% include cloud/admin-ui-login-step.md %}

Author: ```suggestion
{:.procedure}
To set up GeoIp handling:

{% include cloud/admin-ui-login-step.md %}
```

1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.

> 
> 1. In the Magento Admin _Fastly Configuration_ section,

Author: ```suggestion
```

1. Scroll down and select **Yes** to **Enable GeoIP**. Additional configuration options display.

1. For GeoIP Action, select if the visitor is automatically redirected with **Redirect** or provided a list of stores to select from with **Dialog**.

1. For **Country Mapping**, click **Add** to enter a two-letter country code to map with a specific Magento store from a list. For a list of country codes, see [this site](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).

   ![Add GeoIP country maps]({{ site.baseurl }}/common/images/cloud/cloud_fastly-geo-code.png)

1. Click **Save Config** at the top of the page.

1. After page reload, click **Upload VCL to Fastly** in the *Fastly Configuration* section.

Fastly also provides a series of [geolocation-related VCL features](https://docs.fastly.com/guides/vcl/geolocation-related-vcl-features) for customized geolocation coding.

## Enable Fastly Edge modules

Fastly Edge Modules is a flexible framework that allows definition of UI components and associated VCL code through a template. These modules make it easy to customize and extend the Fastly service configuration through the user interface instead of using custom VCL snippets.

> Edge modules allow you to enable specific functionality like CORS headers, Magento Cloud Sitemap rewrites, and to configure integration between your Magento store and other CMSs or backends.

Author: ```suggestion
Edge modules allow you to enable specific functionality like CORS headers, Magento Cloud Sitemap rewrites, and to configure integration between your Magento store and other CMSs or back ends.
```

You must turn on the _Enable Fastly Edge modules_ option to access the Edge Modules menu to view, configure, and manage the available Edge modules.

> ## Configure backends and Origin shielding

Author: ```suggestion
## Configure back ends and Origin shielding
```

> Backend settings provide fine tuning for Fastly performance with Origin shielding and timeouts. A _backend_ is a specific location (IP or domain) with configured Origin shield and timeout settings for checking and providing cached content.

Author: ```suggestion
Back-end settings provide fine tuning for Fastly performance with Origin shielding and timeouts. A _backend_ is a specific location (IP or domain) with configured Origin shield and timeout settings for checking and providing cached content.
```

_Origin shielding_ routes all requests for your store to a specific Point of Presence (POP). When a request is received, the POP checks for cached content and provides it. If it is not cached, it continues to the Shield POP, then to the Origin server which caches the content. The shields reduce traffic directly to the origin.

The default Fastly VCL code specifies default values for Origin shielding and timeouts for your {{ site.data.var.ece }} sites. We recommend using the default values. In some case, you might need to modify the default values. For example, if you are getting a lot of time to first byte (TTFB) errors, you might need to adjust the _first byte timeout_ value.

 {:.bs-callout-info}

> If you need to integrate additional backends into your site such as a backend to serve blog content from a [Wordpress]({{ site.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html) site, you must customize your Fastly service configuration to add the backend and handle the redirects from your {{ site.data.var.ee }} store to the Wordpress backend. For details, see [Fastly Edge Modules - Other CMS/Backend integration](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/Edge-Modules/EDGE-MODULE-OTHER-CMS-INTEGRATION.md) in the Fastly module documentation.

Reviewer: Add missing comma.

```suggestion
If you need to integrate additional backends into your site, such as a backend to serve blog content from a [Wordpress]({{ site.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html) site, you must customize your Fastly service configuration to add the backend and handle the redirects from your {{ site.data.var.ee }} store to the Wordpress backend. For details, see [Fastly Edge Modules - Other CMS/Backend integration](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/Edge-Modules/EDGE-MODULE-OTHER-CMS-INTEGRATION.md) in the Fastly module documentation.
```

> If you need to integrate additional backends into your site, such as a backend to serve blog content from a [Wordpress]({{ site.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html) site, you must customize your Fastly service configuration to add the backend and handle the redirects from your {{ site.data.var.ee }} store to the Wordpress backend. For details, see [Fastly Edge Modules - Other CMS/Backend integration](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/Edge-Modules/EDGE-MODULE-OTHER-CMS-INTEGRATION.md) in the Fastly module documentation.

Author: ```suggestion
If you need to integrate additional back ends into your site, such as a back end to serve blog content from a [Wordpress]({{ site.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html) site, you must customize your Fastly service configuration to add the back end and handle the redirects from your {{ site.data.var.ee }} store to the Wordpress back end. For details, see [Fastly Edge Modules - Other CMS/Backend integration](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/Edge-Modules/EDGE-MODULE-OTHER-CMS-INTEGRATION.md) in the Fastly module documentation.
```

{:.procedure}

> To review the backend settings configuration:

Author: ```suggestion
To review the back end setting configuration:
```

{% include cloud/admin-ui-login-step.md %}

1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.

1. Expand the **Fastly Configuration** section.

> 1. Expand **Backend settings** and click the gear to check the default backend. A modal opens that shows current settings with options to change them.

Author: ```suggestion
1. Expand **Backend settings** and click the gear to check the default back end. A modal opens that shows current settings with options to change them.
```

>    ![Modify the backend]({{ site.baseurl }}/common/images/cloud/cloud_fastly-backend.png){:width="600px"}

Author: ```suggestion
   ![Modify the back end]({{ site.baseurl }}/common/images/cloud/cloud_fastly-backend.png){:width="600px"}
```

> 1. Select the **Shield** location (or datacenter) closest to your Cloud service region.
>    For example, if your project is hosted on AWS, Staging is on the west coast of the United States (us-west-1), select the `sjc-ca-us` Fastly shield location. This is the POP that provides caching services.

Reviewer: Need an extra line.
Also, the "For example" sentence reads a little odd to me with a comma after AWS and a comma after US. Maybe check the sentence to make sure it reads okay. I had to stop a few times, because for me it starts with an "If/then" and then it has a "select action"...yet, they all run together in one sentence. I would separate the two with something a little stronger...like the following suggestion:

```suggestion
1. Select the **Shield** location (or datacenter) closest to your Cloud service region.

   For example, if your project is hosted on AWS, then the Staging is on the west coast of the United States (`us-west-1`); therefore, you would select the `sjc-ca-us` Fastly shield location. This is the POP that provides caching services.
```

   -  ap-east-1 => hongkong-hk
   -  ap-northeast-1 => tyo-tokyo-jp, hnd-tokyo-jp
   -  ap-northeast-2 => tyo-tokyo-jp, hnd-tokyo-jp
   -  ap-southeast-1 => singapore-sg
   -  ap-southeast-2 => sydney-au
   -  ap-south-1 => singapore-sg
   -  ca-central-1 => yul-montreal-ca, iad-va-us, dca-dc-us, bwi-va-us
   -  eu-central-1 => frankfurt-de, hhn-frankfurt-de
   -  eu-north-1 => stockholm-bma
   -  eu-west-1 => london-uk, london_city-uk
   -  eu-west-2 => london-uk, london_city-uk
   -  eu-west-3 => cdg-par-fr
   -  sa-east-1 => gru-br-sa
   -  us-east-1 => iad-va-us, dca-dc-us, bwi-va-us
   -  us-east-2 => iad-va-us, dca-dc-us, bwi-va-us
   -  us-west-1 => sjc-ca-us, pao-ca-us
   -  us-west-2 => sea-wa-us

1. Modify the timeout values (in microseconds) for the connection to the shield, time between bytes, and time for the first byte. We recommend keeping the default timeout settings.

1. Optionally, select to **Activate the backend and Shield after editing or saving**.

1. Click **Upload** to save your changes and upload them to the Fastly servers.

1. In the Magento Admin, click **Save Config**.

> For more information, see the Magento 2 [Backend settings guide](https://github.com/fastly/fastly-magento2/blob/21b61c8189971275589219d418332798efc7db41/Documentation/Guides/BACKEND-SETTINGS.md) in the Fastly module documentation.

Author: ```suggestion
For more information, see the [Backend settings guide](https://github.com/fastly/fastly-magento2/blob/21b61c8189971275589219d418332798efc7db41/Documentation/Guides/BACKEND-SETTINGS.md) in the Fastly module documentation.
```

{:.bs-callout-info}
For Pro plan projects, you must submit a Magento support ticket to update the Fastly domain configuration for your project.

{:.procedure}
To manage Fastly domain configuration from the Magento Admin:

{% include cloud/admin-ui-login-step.md %}

1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.

1. In the Magento Admin _Fastly Configuration_ section, click **Domains**.

1. Click **Manage Domains** to open the Domains page.

1. Add the top-level and subdomain names for the stores in the Cloud environment.

   You can only specify domains that have already been added to your Cloud infrastructure configuration.

   ![Add Fastly domain configuration for Starter]({{ site.baseurl }}/common/images/cloud/cloud-fastly-starter-activate-domain.png){:width="650px"}

1. Click **Activate** to update the Fastly domain configuration.

## Enable Maintenance Mode

Use the _Maintenance Mode_ option to allow administrative access to your site from specified IP addresses while returning an error page for all other requests.

{:.procedure}

> 

Reviewer: remove blank line

> 

Author: ```suggestion
```

> 

Author: ```suggestion
```
